includes/Discovery.php

```diff
@@ -306,14 +311,26 @@ public static function get_publisher_logo() {
 		return apply_filters( 'web_stories_publisher_logo', $logo_image_url );
 	}
 
+
+	/**
+	 * Publisher logo placeholder for static content output which will be replaced server-side.
+	 *
+	 * Uses a fallback logo to always create valid AMP in FE.
+	 *
+	 * @return string
+	 */
+	public function get_publisher_logo_placeholder() {
+		return WEBSTORIES_PLUGIN_DIR_URL . 'assets/images/fallback-wordpress-publisher-logo.png';
```

Comment:
Should we have a filter here?

Reply:
This is just the fallback if there is no publisher logo, as a last resort. If someone wants to override the publisher logo, they can use the `web_stories_publisher_logo` filter.